src/Components/Footer/Footer.css

```diff
@@ -0,0 +1,121 @@
+@import 'https://fonts.googleapis.com/css2?family=Titillium+Web:ital,wght@0,200;0,300;0,400;0,600;0,700;0,900;1,200;1,300;1,400;1,600;1,700&display=swap';
```

Comment:
you don't need from line 1 to 110. they are in the global css file 

and i can see you fix the   justify-content: center;  i was talking about

Reply:
![Screenshot 2024-09-14 at 16 17 44](https://github.com/user-attachments/assets/bd9e3aa7-bc4a-48fb-99aa-837ad435e0ae)

Reply:
its alright inside the file. git just open @use

Reply:
yes that can work but in this case, the global css will need to be added to every components or page css  but what am thinking is you can create a app.scss file and add the global scss link to it and link that file to the app.js file  , this way we turn to apply global css to all component and pages

Reply:
i think this fix should be in your Ruslan ticket  SPF2-2

Reply:
Can do that

Reply:
thanks man

Reply:
> ![Screenshot 2024-09-14 at 16 17 44](https://private-user-images.githubusercontent.com/90725405/367524627-bd9e3aa7-bc4a-48fb-99aa-837ad435e0ae.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjYzMjg4ODUsIm5iZiI6MTcyNjMyODU4NSwicGF0aCI6Ii85MDcyNTQwNS8zNjc1MjQ2MjctYmQ5ZTNhYTctYmM0YS00OGZiLTk5YWEtODM3YWQ0MzVlMGFlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTE0VDE1NDMwNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWNiN2FkZWQyY2FiNTc3YmVmNTkzY2JmMjg4YzQwMzUyYWRkZTMxYzdjNTgyZmJhYjY2NGVjMjAyMjk4YjBlOTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.a2S_OuF2yuzgkWHG4B6_dDZrni69JSrsKVPZeA7F5HY)

@murasakiaura  you only need this code of css

Reply:
nice work your work can be approved now lol